src/Microsoft.Extensions.Options.ConfigurationExtensions/OptionsServiceCollectionExtensions.cs

```diff
@@ -23,6 +23,7 @@ public static IServiceCollection Configure<TOptions>(this IServiceCollection ser
             }
 
```

Comment:
The name of this class can be as ugly as you want, noone uses it, its a static placeholder class for extensions... so go nuts :)